static/admin/preview-pane.css

```diff
@@ -0,0 +1,103 @@
+@import url("./normalize.css");
+@import url("./bootstrap.css");
```

Comment:
I would consider rendering the App Shell rather than trying to reproduce the same CSS styles as N360 within the preview.